docs/user-guide/creating-application/git-material.md

```diff
@@ -1,18 +1,18 @@
 # Git Material
 
-Git material is used to pull your application source code during the CI step. Inside git material when you click on `Add material` you will see three options as shown below:
+Git material is used to pull your application source code during the CI step. Select `Git Material` section of the  `App Configuration`. Inside `Git Material` when you click on `Add Git Material` you will see three options as shown below:
 
-1. Select Provider
+1. Git Account
 2. Git Repo URL
 3. Checkout Path
 
 Devtron also supports multiple git repositories in a single deployment. We will discuss this in detail in the multi git option [below](git-material.md#4-multi-git).
 
-![](../../.gitbook/assets/move1%20%283%29.gif)
+![](../../.gitbook/assets/create-app-git-account.gif)
 
-## 1. Select Provider
+## 1. Git Account
 
-In the provider section, you have to select the git provider of your code repository like- Github, Gitlab, and Bitbucket, etc. If you are using a private git repository, you can configure your git provider via [git provider config](../git-provider-config/).
+In this section, you have to select the git provider of your code repository like- Github, Gitlab, and Bitbucket, etc. If you are using a private git repository, you can configure your git provider via [git provider config](../git-provider-config/).
```

Comment:
heading of this section is git account but we ask people to configure git provider